### ISSUE TITLE
Questiona about Multi gpus using estimator in tensorflow 1.9

### ISSUE THREAD

**Issue by LZDSJTU** [tensorflow/tensorflow#21398]
Now I am using estimator to train models in multi gpus.

When I used the original sess.run() and trained in multi gpus before, I found that the batchnorm statistics cannot be shared between multi gpus and had a bad effect on the evaluation result.

So I change the code to tensorflow 1.9 and used estimator to train. 

Estimator can support a distributed multi-server environment without changing your model I am not sure if estimator support sharing the batchnorm statistics between multi gpus. And I think this is very important. 

So I want to make sure about this.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by drpngx** [comment]
@martinwicke any comment on batchnorm with distributed estimator?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by bhack** [comment]
Do you mean synchronized? https://github.com/tensorflow/tensorflow/issues/18222
